src/bankingAccount/Main.java

```diff
@@ -1,35 +1,221 @@
 package bankingAccount;
 
 
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
 public class Main {
     public static void main(String[] args)
     {
+        Scanner input = new Scanner(System.in);
         //new bank with system properties
         Bank bank = new Bank();
+        System.out.println("Hello Sir, due to launching our bank and you are our first client you should create account to store it in our database and it's account number = 1");
+
+        //index for account no that increment in each account creation
+        int index = 1;
+
+        System.out.println("Please enter your name:");
+        String accountHolderName = input.nextLine();
+
+        System.out.println("Please enter your initial balance:");
+        double accountBalance = 0.0;
+        boolean validInput = false;
+        //loop and checking if the entered balance is valid balance or not
+        while (!validInput) {
+            try {
+                accountBalance = input.nextDouble();
+                input.nextLine(); // consume the newline character
+                validInput = accountBalance >= 0;
+                if (!validInput) {
+                    System.out.println("Please enter a valid positive balance");
+                }
+            } catch (InputMismatchException e) {
+                System.out.println("Invalid input. Please enter a valid balance");
+                input.nextLine(); // consume the invalid input
+            }
+        }
+
+
+
+
+        // variable store the value of account no that initiated by 1
+        String accountNumber = String.valueOf(index);
+
+        System.out.println("Please enter the type of your account\n1- Saving Account\n2- Checking Account");
+        int accountType = input.nextInt();
+        input.nextLine(); // consume the newline character
+
+        // condition for validating on the choosen account type
+        if (accountType == 1) {
+            bank.createSavingAccount(accountHolderName, accountNumber, accountBalance);
+            System.out.println("Account created successfully");
+        }
+        else if (accountType == 2) {
+            bank.createCheckingAccount(accountHolderName, accountNumber, accountBalance);
+            System.out.println("Account created successfully");
+        }
+        else
+            System.out.println("Wrong choice, enter a valid choice next time");
+
+        // review the info of the first account
+        bank.reviewAccountInformation(accountNumber);
+        System.out.println("-----------------------\n");
+        System.out.println("Account created successfully");
+
```

Comment:
I think it would be better to place this inside the below do-while, as we already have "create account" as an option.